### PR TITLE
Fixes #9905:  java.lang.InternalError when calling the internal API on Redhat because of invalid version of nss

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9905

### CODE REVIEW

File: rudder-jetty/SOURCES/patches/jetty/jetty-init-softokn-version-check.patch [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+@@ -483,6 +483,14 @@
+     UMASK="0007"
+     echo "Setting umask to ${UMASK}"
+ 
++    # Check if there is a mismatch in nss version, see http://www.rudder-project.org/redmine/issues/9905
++    nss_ver=$(rpm -qa nss | cut -d - -f 2)
++    softokn_ver=$(rpm -qa nss-softokn | cut -d - -f 3)
++    if [ "${nss_ver}" != "${softokn_ver}" ]
++    then
++      echo "NSS version mismatch, your nss-softokn version (${softokn_ver}) sould match your ns version (${nss_ver})"
```
Comment (amousset) [inline rudder-jetty/SOURCES/patches/jetty/jetty-init-softokn-version-check.patch]: should
nss

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Pull request merged**